amplify/backend/function/teamgetOUs/src/index.py

```diff
@@ -16,7 +16,7 @@ def getOUs(id):
         )
         results = response["OrganizationalUnits"]
         while "NextToken" in response:
-            response = client.list_accounts(NextToken=response["NextToken"])
+            response = client.list_organizational_units_for_parent(NextToken=response["NextToken"])
```

Comment:
I think you also need  `ParentId=id` in params. 
I tried without and it and got a param exception:

```
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Missing required parameter in input: "ParentId"
```

Reply:
Yes indeed; Thanks for checking again. I can't test this properly in my environment. I'll try to setup another env on which I can test this properly. For now, if you copy the code in this file to your lambda, it works fine for you?